tests/CMakeLists.txt

```diff
@@ -226,7 +222,6 @@ set_tests_properties(http_plugin_test PROPERTIES TIMEOUT 100)
 set_property(TEST http_plugin_test PROPERTY LABELS nonparallelizable_tests)
 
 add_test(NAME plugin_http_api_test COMMAND tests/plugin_http_api_test.py WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
-set_tests_properties(plugin_http_api_test PROPERTIES TIMEOUT 40)
```

Comment:
I'm not sure the `TIMEOUT` should be completely removed.  I'd be okay with increasing the timeout a bit if it is requiring more time after some changes, but prefer not giving the test carte blanche to run forever.